cmd/xpki-tool/cli/crl.go

```diff
@@ -76,12 +77,12 @@ func (a *CRLFetchCmd) Run(ctx *Cli) error {
 	}
 	for _, crt := range list {
 		if len(crt.CRLDistributionPoints) < 1 {
-			logger.Infof("CRL DP is not present: CN=%q\n", crt.Subject.String())
+			logger.KV(xlog.INFO, "CRL DP is not present: CN", crt.Subject.String())
```

Comment:
logger.KV(xlog.DEBUG, "reason", "CRL DP is not present",  "CN", crt.Subject.String())`